.github/actions/veracode/source_clear.sh

```diff
@@ -4,10 +4,7 @@ echo "=========================== Starting SourceClear Script ==================
 PS4="\[\e[35m\]+ \[\e[m\]"
 set +e -v -x
 
-mvn -B -q clean install \
-    ${SRCCLR_INSTALL_DEFAULT_OPTIONS} ${SRCCLR_INSTALL_OPTIONS} \
-    com.srcclr:srcclr-maven-plugin:scan \
-    -Dcom.srcclr.apiToken=${SRCCLR_API_TOKEN} > scan.log
+srcclr scan > scan.log
```

Comment:
Does this tool respect the https://github.com/Alfresco/alfresco-azure-connector/blob/master/srcclr.yml files that are scattered across our repositories? If not, this will be a significant issue.

Reply:
https://docs.veracode.com/r/Veracode_SCA_Agent_Commands#scan

should be in the same folder of srccclr

Reply:
Yes it does, in fact it will require the "srcclr-install-options" custom values to be placed in that same file to work, so will require a slight rework for a few projects (HxInsight, ESC, ATS, maybe Audit component "coming soon") that were using this option until now.